guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
++ client.shard.respawnAll({ shardDelay: 5000, respawnDelay: 500, timeout: 30000 });
+```
+
+### ShardingManager
+
+#### ShardingManager#broadcastEval
+
+The `ShardingManager#broadcastEval` method no longer accepts a string, instead expecting a function. See `ShardClientUtil#broadcastEval`.
+
+#### ShardingManager#spawn
+
+The options for the `ShardingManager#spawn` method are now an object instead of separate params. In addition the `spawnTimeout` param has been renamed to `timeout`.
+
+This means the user no longer needs to pass defaults to fill each positional param.
+
+```diff
+- manager.spawn('auto', 5500, 30000);
++ manager.spawn({ amount: 'auto', delay: 5500, timeout: 30000 });
+```
+
+#### ShardingManager#respawnAll
+
+The options for the `ShardingManager#respawnAll` method are now an object instead of separate params. In addition the `spawnTimeout` param has been renamed to `timeout`.
+
+This means the user no longer needs to pass defaults to fill each positional param.
+
+```diff
+- manager.respawnAll(5000, 500, 30000);
++ manager.respawnAll({ shardDelay: 5000, respawnDelay: 500, timeout: 30000 });
+```
+
+### User
+
+#### User#locale
+
+The `User.locale` property has been removed, as this property is not exposed to bots.
+
+### UserFlags
+
+The deprecated UserFlags `DISCORD_PARTNER` and `VERIFIED_DEVELOPER` / `EARLY_VERIFIED_DEVELOPER` have been removed in favor of their renamed versions.
+
+```diff
+- user.flags.has(UserFlags.FLAGS.DISCORD_PARTNER)
++ user.flags.has(UserFlags.FLAGS.PARTNERED_SERVER_OWNER)
+
+- user.flags.has(UserFlags.FLAGS.VERIFIED_DEVELOPER)
++ user.flags.has(UserFlags.FLAGS.EARLY_VERIFIED_BOT_DEVELOPER)
+```
+
+The new flag `DISCORD_CERTIFIED_MODERATOR` has been added.
+
+### Util
+
+Shortcuts to Util methods which were previously exported at the top level have been removed.
+
+#### Util#convertToBuffer
+
+#### Util#str2ab
+
+Both were removed in favor of Node built-in Buffer methods.
+
+#### Util#resolveString
+
+The `Util#resolveString` method has been removed. Discord.js now enforces that users provide strings where expected rather than resolving one on their behalf.
+
+## Additions
+
+### ActivityTypes
+
+A new activity type `COMPETING` has been added.
+
+### ApplicationCommand
+
+Provides API support for Slash Commands.
+
+### ApplicationCommandManager
+
+Provides API supporting for creating, editing and deleting Slash Commands.
+
+### ApplicationCommandPermissionsManager
+
+Provides API support for creating, editing and deleting permission overwrites on Slash Commands.
+
+### ApplicationFlags
+
+Provides an enumerated bitfield for `ClientApplication` flags.
+
+### BaseGuild
+
+The new `BaseGuild` class is extended by both `Guild` and `OAuth2Guild`.
+
+### ButtonInteraction
+
+Provides gateway support for a `MessageComponentInteraction` coming from a button component.
+
+### Channel
+
+#### Channel#isText()
+
+Checks and typeguards if a channel is Text-Based ("dm", "text", "news").
```

Comment:
I think we should word those differently, in case https://github.com/discordjs/discord.js/pull/6035 merges before v13.

Also [needs to include threads](https://discord.js.org/#/docs/main/master/class/Channel?scrollTo=isText).